docs/quick-start/transmitters/tx-prep.md

```diff
@@ -80,13 +80,13 @@ For Radios equipped with Internal ExpressLRS modules, like the RadioMaster Zorro
 </figure>
 
 !!! Note
-    iFlight Commando 8 is NOT using an internal RF module for its ExpressLRS units, unlike the RadioMaster Zorro, RadioMaster TX16S MK2, RadioMaster TX12 MK2, Jumper T-lite V2, Jumper T-pro or the BetaFPV Lite Radio 3 Pro.
+    The iFlight Commando 8 is not using an internal RF module for its ExpressLRS units, unlike the RadioMaster Zorro, RadioMaster TX16S MK2, RadioMaster TX12 MK2, Jumper T-lite V2, Jumper T-pro or the BetaFPV Lite Radio 3 Pro.
```

Comment:
All CAPS in "NOT" is there to drive a point. To put emphasis. To tell users in a factual manner.
That's what I'm hoping.